scripts/integration-tests/utils/local-registry.sh

```diff
@@ -3,14 +3,14 @@
 # Copied from https://github.com/facebook/create-react-app/blob/053f9774d3f592c17741d2a86de66a7ca58f90c0/tasks/local-registry.sh
 
 custom_registry_url=http://localhost:4873
-default_verdaccio_package=verdaccio@~4.3.3
+default_verdaccio_package=verdaccio@~4.10.0
 
 function startLocalRegistry {
   # Start local registry
   tmp_registry_log=`mktemp`
   echo "Registry output file: $tmp_registry_log"
   (cd && nohup npx ${VERDACCIO_PACKAGE:-$default_verdaccio_package} -c $1 &>$tmp_registry_log &)
-
+  npm install --global verdaccio-memory
```

Comment:
I'd pin `verdaccio-memory@9.7.2` . There are few major coming soon that might affect the pipeline.